sdk/testing/stepwise/helpers.go

```diff
@@ -0,0 +1,51 @@
+package stepwise
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+)
+
+const pluginPrefix = "vault-plugin-"
+
+// CompilePlugin is a helper method to compile a source plugin
+// TODO refactor compile plugin input and output to be types
```

Comment:
Since all non-error return values are strings, might be good to add a comment around them so consumers don't need to look into the source to figure this out.